extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -227,30 +227,28 @@ function Ach_UserPreferences( $user, &$preferences ) {
 /*
  * Used in the mantainance/wikia/takeAchievementsRankingSnapshot.php script
  */
-function Ach_TakeRankingSnapshot($force = false) {
+function Ach_TakeRankingSnapshot( $force = false ) {
 	global $wgCityId;
 	$dbw = WikiFactory::db( DB_MASTER );
 
-	$res = $dbw->select('ach_ranking_snapshots', array('date'), array('wiki_id' => $wgCityId), __METHOD__);
+	$res = $dbw->select( 'ach_ranking_snapshots', array( 'date' ), array( 'wiki_id' => $wgCityId ), __METHOD__ );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wiki_id" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "ach_ranking_snapshots" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)